src/core/atomicdex/utilities/cpprestsdk.utilities.cpp

```diff
@@ -54,3 +54,23 @@ handle_exception_pplx_task(pplx::task<void> previous_task)
 #endif
     }
 }
+
+pplx::task<std::filesystem::path> download_file(const t_http_client_ptr& client, const std::string& url, const std::filesystem::path& output_location)
+{
+    SPDLOG_DEBUG("Start downloading {} to {}", url, output_location);
```

Comment:
You cannot log a `fs::path` on every platform atm, let's use `c_str()` and not `string()` since we use `boost::filesystem` on osx.